openadapt/crud.py

```diff
@@ -22,10 +29,17 @@
 memory_stats = []
 
 
+def _insert(
+    event_data: dict, table: sa.Table, buffer: list[Any] | None = None
```

Comment:
```
buffer: list[dict[str, Any]]
```

Reply:
Addressed in https://github.com/OpenAdaptAI/OpenAdapt/pull/289/commits/ea5e2a6ac19b1a3ea2ce9b5c89fb410dfa0bb171